### ISSUE TITLE
FreeBSD build error: no SSL support

### ISSUE THREAD

**Issue by kbelyavs** [tarantool/tarantool#3750]
On 1.10, FreeBSD 11.1 cmake fails with following error:
-- Performing Test CURL_SUPPORTS_SSL - Failed

**Comment by Totktonada** [comment]
Tried to build from source on FreeBSD 11.2 and all seems to be fine. Are you have any other information how to reproduce the issue?

**Comment by Totktonada** [comment]
Reproduced on FreeBSD 10.4.

**Comment by Totktonada** [comment]
```
/usr/bin/cc  -Wno-unknown-pragmas -fexceptions -funwind-tables -fno-common -DCURL_SUPPORTS_SSL    CMakeFiles/cmTC_d99a3.dir/src.c.o  -o cmTC_d99a3 -Wl,-rpath,/usr/local/lib /usr/local/lib/libcurl.so /usr/lib/libssl.so /usr/lib/libcrypto.so -lpthread -ldl 
/usr/bin/ld: cannot find -ldl
cc: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by locker** [comment]
Fixed by 46e4c702b09dc3ad03d528d5bf8447a41b8c9d6d
